packages/http/package.json

```
@@ -42,6 +42,10 @@
     "./experimental/typekit": {
       "types": "./dist/src/experimental/typekit/index.d.ts",
       "default": "./dist/src/experimental/typekit/index.js"
+    },
+    "./experimental/merge-patch": {
+      "types": "./dist/src/experimental/merge-patch/index.d.ts",
+      "default": "./dist/src/experimental/merge-patch/index.js"
```

```suggestion
      "import": "./dist/src/experimental/merge-patch/index.js"
```
for new imports we should try to use the recommended syntax for esm